Lib/urllib/parse.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+__all__ = ['ParseResult', 'SplitResult', 'parse_qs', 'parse_qsl', 'urldefrag',
+           'urljoin', 'urlparse', 'urlsplit', 'urlunparse', 'urlunsplit',
+           'quote', 'quote_plus', 'unquote', 'unquote_plus',
+           'unquote_to_bytes', 'urlencode', 'splitquery', 'splittag',
+           'splituser', 'splitvalue', 'uses_fragment', 'uses_netloc',
+           'uses_params', 'uses_query', 'uses_relative', 'unwrap']
+
+from urllib import (quote as quote_from_bytes, unquote_plus,
+                    unquote as unquote_to_bytes, urlencode, splitattr,
+                    splithost,
+                    splitpasswd, splitport, splitquery, splittag, splittype,
+                    splituser, splitvalue, unwrap)
+from urlparse import (__doc__, ParseResult, SplitResult, parse_qs, parse_qsl,
+                      urldefrag, urljoin, urlparse, urlsplit, urlunparse,
+                      urlunsplit, uses_fragment, uses_netloc, uses_params,
+                      uses_query, uses_relative)
+
+# Functions modified from Python 3's urllib.
```

Comment:
How about some tests for these functions?

Reply:
Added.